grails-app/assets/javascripts/project-finder.js

```diff
@@ -412,13 +422,15 @@ function ProjectFinder(config) {
             projectId: selectedProjectId,
             q: query,
             queryList: queryList,
-            queryText: this.getQuery(true)
+            queryText: this.getQuery(true),
+            // sending the value for logging use
+            excludeNationwide: excludeNationwide
```

Comment:
in parseHash function, make sure you check for excludeNationwide and set checkbox value if it has been set.